### PR TITLE
Using webpack aliases in simorgh

### LINKED ISSUE #2567: Add webpack directory aliasing
**Is your feature request related to a problem? Please describe.**
As simorgh gets bigger we are ending up with imports like `'../../../../lib/analyticsUtils'` which isnt very readable at all.

**Describe the solution you'd like**
Add webpack aliases so rather than `'../../../../lib/analyticsUtils'` you can do `'@lib/analyticsUtils'`

Possible other ones include `@components` `@containers` `@contexts` `@testHelpers` `@app` `@server` 

**Testing notes**
[Tester to complete]

Dev insight: Just a sanity check that it builds fine, pages load all correct stuff etc

**Additional context**

### PR DESCRIPTION
**Pull request by oluoluoxenfree**
Resolves #2567 

**Overall change:** 

Uses webpack directory aliasing previously set up in #3318 

**Code changes:**

- _A bullet point list of key code changes that have been made._
- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: cypress/integration/specialFeatures/cookieBanner/testsForAMPOnly.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import config from '../../../support/config/services';
-import appConfig from '../../../../src/app/lib/config/services';
+import appConfig from '#app/lib/config/services';
```
Comment (jamesbhobbs) [inline cypress/integration/specialFeatures/cookieBanner/testsForAMPOnly.js]:
```suggestion
import appConfig from '#lib/config/services';
```

File: cypress/integration/specialFeatures/cookieBanner/testsForCanonicalOnly.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import appConfig from '../../../../src/app/lib/config/services';
+import appConfig from '#app/lib/config/services';
```
Comment (jamesbhobbs) [inline cypress/integration/specialFeatures/cookieBanner/testsForCanonicalOnly.js]:
```suggestion
import appConfig from '#lib/config/services';
```

File: cypress/support/helpers/useAppToggles.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import togglesConfig from '../../../src/app/lib/config/toggles';
+import togglesConfig from '#app/lib/config/toggles';
```
Comment (jamesbhobbs) [inline cypress/support/helpers/useAppToggles.js]:
```suggestion
import togglesConfig from '#lib/config/toggles';
```

**Review by jamesbhobbs — APPROVED** [review]

**Review by jamesbhobbs — CHANGES_REQUESTED** [review]
Found those in storybook and cypress need reverting, they don't share the same webpack setup so cannot use these aliases at all

**Comment by jamesbhobbs** [comment]
FYI @hindsc52 @dr3 

Ok so problems in order:

1. At first glance this looks pretty good. But the build passes when it absolutely shouldn't! Storybook won't run, the only e2es that still run are those missed in the PR (cypress/integration/application/index.js)
2. Some files are missed as proved by some e2es still working in point 1
3. Some files changed use app when they could directly use lib. I fixed a couple and later realised I missed some more, but this doesn't really matter because...
4. This approach cannot work at all. The webpack config is for our app, the webpack config is not shared in cypress, 3rdPartyCypress or storybook - so every single alias reference in there fails - without erroring the build. This is the cause of point 1. We cannot even remove change to storybook because the story files themselves have aliases that it cannot understand. If we reverted these back them I'm not certain the components themselves could render because it'll still try to render those components in storybook with the unresolvable aliases. So we'd then have to not use aliases in components/containers or we'd have to extend the webpack config to work for storybook. But is another setup task to be done before this.

Is all this worth the aliases? I thought this'd be easy when I wrote the issue. I'm tempted to revert the addition of aliases and not do this at all.

Horrifying that such an innocuous seeming PR can cause a confusing thing as a passing build with nearly everything outside the main app failing.

PS how can we avoid all tests not running and the build still passing in future PRs? That seems much more important to fix that anything we're trying to do here.

PPS @oluoluoxenfree don't let this get you down, we all missed the dragons herein when we approved the preparatory PR - maybe Chris or Drew have a quickfix, but I can't think of one.

**Comment by oluoluoxenfree** [comment]
@benjaminhobbs can we not do something like this?

https://github.com/storybookjs/storybook/issues/3339

**Comment by thekp** [comment]
I am closing this PR, to continue it here : https://github.com/bbc/simorgh/pull/3707

**Pull request closed without merging**